docs/explanation/high-availability.md

```diff
@@ -0,0 +1,40 @@
+# High availability
+High availability (HA) is a characteristic of a system that aims to ensure an agreed level of operational performance, usually uptime, for a higher than normal period. In this document we will discuss different possible configurations for High availability supported by the `hacluster` subordinate charm.
+
+# Active/active
+An active-active configuration consists of two or more Haproxy units that reply to incoming requests at the same time. Increasing the number of haproxy charm units can be done either during deployment with the `-n` flag or by using the `juju add-unit` command: 
```

Comment:
minor: I think the official documentation uses "HAProxy".

Reply:
thanks, I updated it